src/ui/panes/album_art.rs

```diff
@@ -140,7 +140,7 @@ impl Pane for AlbumArtPane {
             UiEvent::ModalClosed if is_visible => {
                 self.is_modal_open = false;
 
-                if self.fetch_needed {
+                if self.fetch_needed || context.config.album_art.method == ImageMethod::Block {
```

Comment:
This is not the correct solution, the same problem exists in other places and this fixes only the modal. The modal closing is just easiest to reproduce.
```suggestion
                if self.fetch_needed {
```